app/src/components/learningJourney/index.jsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import LearningJourneyTile from "./LearningJourneyTile";
+
+function LearningJourney() {
+  const LearningJourneyData = [
```

Comment:
Pull mockdata out of functions, rename to mockXXXXX. I have create a mockdata file also under utils. Can place it inside if u want.